src/components/application_manager/src/commands/mobile/perform_audio_pass_thru_request.cc

```diff
@@ -343,6 +343,30 @@ void PerformAudioPassThruRequest::FinishTTSSpeak(){
   SendHMIRequest(hmi_apis::FunctionID::TTS_StopSpeaking, NULL);
 }
 
+bool PerformAudioPassThruRequest::WaitTTSSpeak() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  uint32_t default_timeout_msec =
+      profile::Profile::instance()->default_timeout();
+  TimevalStruct start_time = date_time::DateTime::getCurrentTime();
+
+  // Waiting for TTS_Speak
+  while (is_active_tts_speak_) {
```

Comment:
@pestOO , Yes, we need wait for TTS.Speak for default timeout in *.ini file. Checked all lua scripts for PerformAudioPassThru.